lib/app.dart

```diff
@@ -122,9 +122,7 @@ class _LanternAppState extends State<LanternApp>
         ChangeNotifierProvider(create: (context) => VPNChangeNotifier()),
         ChangeNotifierProvider(create: (context) => InternetStatusProvider())
       ],
-      child: ChangeNotifierProvider(
```

Comment:
@jigar-f I think this may be another reason why the app has been crashing.. it looks like we are using two ChangeNotifierProviders here? We are already passing an instance to MultiProvider.

Reply:
I might have forgotten to remove that, but I don't think that should crash app.

Reply:
Ok cool, I assume we only need to use a single ChangeNotifierProvider though